src/liquid/components/ld-input/readme.md

```diff
@@ -161,6 +165,10 @@ By default, the `ld-input` component is of [type `text`](https://developer.mozil
 
 ### Type datetime-local
 
+<ld-notice headline="Note" mode="info">
+  Just like the <code>ld-input</code> component of <code>type="date"</code> the <code>type="datetime-local"</code> version acts as a wrapper around its native <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/datetime-local"><code>input</code> counterpart</a>, fully relying on its built-in features and only enhancing it visually. Its date and time format is intentionally <strong>not</strong> configurable via a prop, but instead depends on the user's <abbr title="operating system">OS</abbr> settings.
+</ld-notice>
```

Comment:
This box is basically explaining that it behaves similar to the date input while adding the topic that we do not provide configuration on purpose. I think we should avoid arguing about omitted props and focus on expecting the native behaviors.

Moreover, we should not assume that the user is reading the documentation consecutively. So maybe we should avoid references to previous blocks?

```suggestion
<ld-notice headline="Note" mode="info">
  The <code>ld-input</code> component of <code>type="datetime-local"</code> acts as a wrapper around the native <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/datetime-local"><code>input</code> element of<code>type="datetime-local"</code></a>, fully relying on its built-in features and only enhancing it visually.<br />The component is lightweight, respects the user preferences for display format configured in the user's <abbr title="operating system">OS</abbr> settings, separates presentation format from wire format and is accessible out of the box by offering a familiar <abbr title="user interface">UI</abbr>.
</ld-notice>
```